localstack/services/events/events_starter.py

```diff
@@ -66,18 +66,19 @@ def filter_event(event_pattern, event):
         return True
 
     rule_information = self.events_backend.describe_rule(rule)
-    if rule_information.event_pattern:
-        event_pattern = rule_information.event_pattern._filter
-        if event_pattern and not filter_event(event_pattern, event):
-            return False
+    rule_event_pattern = json.loads(str(rule_information.event_pattern))
```

Comment:
nitpick: better to use `to_str(..)`/`to_bytes(..)` from `utils/common.py` for bytes<>string conversion.
```
     rule_event_pattern = json.loads(to_str(rule_information.event_pattern))
```